coderd/database/dbauthz/dbauthz.go

```diff
@@ -219,7 +219,7 @@ var (
 		Scope: rbac.ScopeAll,
 	}.WithCachedASTValue()
 
-	// See unhanger package.
+	// See reaper package.
 	subjectHangDetector = rbac.Subject{
 		Type:         rbac.SubjectTypeHangDetector,
```

Comment:
Since this only exists as a system in-memory rbac subject it should be safe to rename everything in this file in the subject (incl. the `Type` and the enum value) as well as the `As` function.